### DISCUSSION TITLE
Spacehammer Keycap

### DISCUSSION THREAD

**Discussion by coderberry** [agzam/spacehammer#88] [Show and tell]
I created a 3D printable keycap for Spacehammer. Feel free to use/modify.

https://www.tinkercad.com/things/lOrEvhJvUOs-hammer-keycap

![image](https://user-images.githubusercontent.com/12481/114962960-f1f24900-9e28-11eb-8fda-cfa053fd5cb2.png)

![image](https://user-images.githubusercontent.com/12481/114962894-ccfdd600-9e28-11eb-862b-a21f83bf9346.png)

![image](https://user-images.githubusercontent.com/12481/114963049-1d753380-9e29-11eb-8e09-fd901901eab6.png)

**Comment by jaidetree** [comment]
This is so cool! I’ve got an idea to make a banner for the repo based on the first image, just gotta try a few things. 😁
